nano/node/json_handler.cpp

```diff
@@ -5224,6 +5224,13 @@ void nano::json_handler::work_peers_clear ()
 	response_errors ();
 }
 
+void nano::json_handler::populate_backlog ()
+{
+	node.populate_backlog ();
```

Comment:
I think rather than calling populate backlog directly, it should just trigger one.
I do not think we should be doing heavy processing in the RPC handler.

Reply:
The problem with triggering it to be run in the background is that way it would be possible to starve all `node.workers` and node needs those for realtime processing. This way it is only the rpc that will stop responding for a moment, not the whole node.

Reply:
We could move the backlog population to a dedicated thread and trigger it there (in a dedicated class perhaps), that sounds reasonable to me.